package.json

```diff
@@ -38,12 +38,10 @@
     "typescript": "2.9.2"
   },
   "dependencies": {
+    "diskusage": "1.1.2",
```

Comment:
This library is more like a native addon isn't it? Does this work against Node 12?

Reply:
@VishwasShashidhar Yes, it supports Node 12.